build/Targets/Packages.props

```diff
@@ -66,6 +66,7 @@
     <MicrosoftNuGetBuildTasksVersion>0.1.0</MicrosoftNuGetBuildTasksVersion>
     <MicrosoftPortableTargetsVersion>0.1.2-dev</MicrosoftPortableTargetsVersion>
     <MicrosoftServiceHubClientVersion>1.1.122-rc</MicrosoftServiceHubClientVersion>
+    <MicrosoftTestApexVersion>15.0.26621.11111</MicrosoftTestApexVersion>
```

Comment:
How can we map APEX versions to VS versions? That is when I move to a new version of VS how can I tell what APEX i should be using?

Reply:
@andrewarnott, could you please help?

---
In reply to: [162495630](https://github.com/dotnet/roslyn/pull/22201#discussion_r162495630) [](ancestors = 162495630)